client/client.go

```diff
@@ -60,6 +61,10 @@ func (c *client) Health() error {
 func (c *client) GetData(ctx context.Context, comm []byte) ([]byte, error) {
 	url := fmt.Sprintf("%s/get/0x%x?commitment_mode=simple", c.cfg.URL, comm)
 
+	if c.cfg.Actor != "" {
```

Comment:
One high level comment, the eigenda-proxy should be configured to work correctly most of time. Is it reasonable to make Actor a lower layer concept subject to a specific boundary.

Reply:
wdym?